datamade/how-to
# upgrade gatsby cookiecutter

## Overview

- closes #273

## Testing Instructions
- follow the cookiecutter instructions for creating a new gatsby app (via `docker-compose run --rm cookiecutter -f new-gatsby-app` command)
- follow the instructions in your new gatsby app's readme to build and run the app
- confirm it builds and runs with no problems
